student.rb

```diff
@@ -0,0 +1,13 @@
+# rubocop:disable all
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
+
```

Comment:
- [ ] When initialising a student, you still need to expect name and parent_permission params even though Student class is inheriting from Person. Otherwise you will get argument error. 

```
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
Corrected